packages/cpp-debug/src/browser/memory-widget/memory-options-widget.tsx

```diff
@@ -643,6 +643,10 @@ export class MemoryOptionsWidget extends ReactWidget implements StatefulWidget {
         }
     }
 
+    protected getUserError(_err: unknown): string {
+        return 'There was an error fetching memory with specified address length.';
```

Comment:
I've just been doing some debugging with a colleague, and it seems that the messages that we get is often reasonably helpful, often moreso than this generic message. So perhaps the default behavior should be something like 
```typescript
return err instanceof Error ? err.message : 'There was an error fetching memory.';
```

What do you think of that?